influxdb/meta/collectd.yml

```diff
@@ -0,0 +1,17 @@
+{%- if pillar.influxdb.server is defined %}
+{%- from "influxdb/map.jinja" import server with context %}
+
+{%- if server.admin.get('user', {}).get('enabled', false) and server.http.enabled %}
```

Comment:
What if `http` is not set on `server`? And what if `enabled` is not set on `http`?